### DISCUSSION TITLE
404 when trying to open an attachment

### DISCUSSION THREAD

**Discussion by rodrigolfh** [django-helpdesk/django-helpdesk#1212] [Q&A] [answered]
Whenever I try to open an attachment i get:

![image](https://github.com/user-attachments/assets/49c99677-cf73-46a7-9016-ec17c03d1921)

I see in the terminal : 404 not found

The link points to https://www.site.cl/helpdesk/attachments/red-e-internet-1-0b73063c-a0c8-4b25-a48a-69f273c5df62/5/aclaracion.pdf

And the file is physically located in ls -l /home/user/soporte/soporte/helpdesk/attachments/red-e-internet-1-0b73063c-a0c8-4b25-a48a-69f273c5df62/5/aclaracion.pdf

Thanks in advance

**Comment by Benbb96** [comment]
How is configured your `MEDIA_ROOT` and `MEDIA_URL`?

This depends on the way you make uploaded files accessible. It does not seem to be related to the _django-helpdesk_ package.

**Reply by rodrigolfh** [reply]
After many trials and only errors, this is my current setting:

nginx :  
location /soporte/ {
       root /home/user/soporte/soporte;

settings.py:
MEDIA_ROOT = '/home/user/soporte/soporte/'
MEDIA_URL = '/'

file path in db:
helpdesk/attachments/red-e-internet-1-0b73063c-a0c8-4b25-a48a/6/ANT.pdf

**Reply by Benbb96** [reply]
I believe you need to change the `MEDIA_URL` to `soporte/` in order to point to the location in your nginx configuration :

```
location /soporte/ {
    alias /home/user/soporte/soporte;
}
```

**Comment by rodrigolfh** [comment, marked as answer]
Thanks! I managed to serve the files.
